src/main.controller.js

```diff
@@ -471,12 +476,54 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 					})[0];
 
 					if (field && 'choices' in field) {
-						return field.choices;
+						return escapeChoices(field.choices);
 					}
 				}
 			}
 		};
 
+		/**
+		 * Replaces all "$" symbols for all choices, for a given field.
+		 * This applies to checkbox, dropdown and radio fields.
+		 *
+		 * @param {Object} choices The form field's choices
+		 *
+		 * @return {Object}
+		 */
+		function escapeChoices(choices) {
+			var escaped = {};
+
+			Object.keys(choices).forEach(choice => {
+				if (typeof choice !== 'string') { return; }
+
+				escaped[choice.replace(/\$/g, '*dollar*')] = choices[choice];
+			});
+
+			return escaped;
+		}
+
+		/**
+		 * Conditionally replaces all "$" symbols or "*dollar*" patterns for
+		 * all values in a given configuration.
+		 * This applies to checkbox, dropdown and radio fields.
+		 *
+		 * @param {Object} config The configuration
+		 * @param {boolean} inbound Is the configuration being loaded or saved
+		 *
+		 * @return {Object}
+		 */
+		function escapeValues(config, inbound) {
+			var escape = inbound ? /\$/g : /\*dollar\*/g;
+			var replacement = inbound ? '*dollar*' : '$';
```

Comment:
what happens if you your choice value is actually `*dollar*` ?

Reply:
As of right now, the module will accidentally replace your `*dollar*` with a `$` symbol, so I can take a look at accounting for that.

Reply:
Updated. Let me know what you think.